### DISCUSSION TITLE
Is it possible to add a function button to download the image?

### DISCUSSION THREAD

**Discussion by ghost** [goharbor/harbor#19664] [Ideas]
Because many times, using the save command to save offline is also an operation. It would be more convenient if the page can be clicked directly to complete.
![image](https://github.com/goharbor/harbor/assets/71524776/30414364-1125-44cb-bd2b-842583915d83)

**Comment by zyyw** [comment]
You could use the docker pull command and docker save command to download the image

**Reply by ghost** [reply]
You are right, but I just suggest that it would be better to have this function of clicking the download button, so that it can be used by developers or business personnel who are not familiar with the command line.
